containers/fedora/build.sh

```diff
@@ -35,13 +37,26 @@ else
     exit 1
 fi
 
+if [ $FULL_EMULATION = "true" ]; then
+    VIRT_TYPE="qemu"
+fi
+
+set +x
 FEDORA_PASSWORD=$(uv run get_fedora_password.py)
 PASSWORD_PLACEHOLDER="CHANGE_ME"
 sed -i "s/password: $PASSWORD_PLACEHOLDER/password: $FEDORA_PASSWORD/" user-data
+set -x
 
 echo "Create cloud-init user data ISO"
 cloud-localds $CLOUD_INIT_ISO user-data
 
+# When running on CI, the latest Fedora may not be yet supported by virt-install
+# therefor use a lower one. It should have no influence on the result.
+OS_VARIANT=$NAME
+if [ $FEDORA_VERSION -gt 40 ]; then
```

Comment:
Same here.
You are assuming that the latest available fedora version is  41, so when fedora advances, it will be have to be updated in several places. We better keep the parameter that should be manually changed in one place (in the matrices in [global_config.py](https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/global_config.py)).